docs/operator-manual/user-management/auth0.md

```diff
@@ -69,5 +69,5 @@ data:
 
 <br>
 
-!!! note "Storing Client Secrets"
-    Details on storing your clientSecret securely and correctly can be found on the [User Management Overview page](../../user-management/#sensitive-data-and-sso-client-secrets).
\ No newline at end of file
+!!! **Note** : "Storing Client Secrets"
```

Comment:
@kalyan-vurugonda can you revert this line? it breaks the ReadTheDocs markdown formatting.

![image](https://user-images.githubusercontent.com/350466/189397704-12d03413-e1ae-42dc-a10b-626dbf85a5c9.png)

You can test with `make serve-docs-local`

Reply:
Hi @crenshaw-dev
I made changes as you mentioned above.